configure.ac

```diff
@@ -246,22 +244,25 @@ AS_IF([test "x$with_docs_only" = xyes],
      AC_ARG_ENABLE([params-check],
          AS_HELP_STRING([--disable-params-check],
                         [Disable checking user parameters passed to API, default: NO]),
-         [AC_DEFINE([ENABLE_PARAMS_CHECK], [0])],
-         [AC_DEFINE([ENABLE_PARAMS_CHECK], [1], [Enable checking user parameters])])
-
+         [], 
+         [enable_params_check=no])
+     AS_IF([test "x$enable_param_check" = xyes],
+     	   [AC_DEFINE([ENABLE_PARAMS_CHECK], [0])],
+           [AC_DEFINE([ENABLE_PARAMS_CHECK], [1], [Enable checking user parameters])])
 
      #
      # Enable collecting data to ease debugging
      #
      AC_ARG_ENABLE([debug-data],
                    AS_HELP_STRING([--enable-debug-data],
                                   [Enable collecting data to ease debugging, default: NO]),
-         [
-          AC_DEFINE([ENABLE_DEBUG_DATA], [1], [Enable collecting data])
-          AC_DEFINE([UCT_UD_EP_DEBUG_HOOKS], [1],
-                    [Enable packet header inspection/rewriting in UCT/UD])
-         ],
-         [AC_DEFINE([ENABLE_DEBUG_DATA], [0])])
+		   [], 
+		   [enable_debug_data=no])
+     AS_IF([test "x$enable_debug_data" = xyes],
+           [AC_DEFINE([ENABLE_DEBUG_DATA], [1], [Enable collecting data])
+            AC_DEFINE([UCT_UD_EP_DEBUG_HOOKS], [1],
+                      [Enable packet header inspection/rewriting in UCT/UD])],
+           [AC_DEFINE([ENABLE_DEBUG_DATA], [0])])
```

Comment:
shall we also define UCT_UD_EP_DEBUG_HOOKS to 0?

Reply:
yes